### ISSUE TITLE
Check that emails are being sent...

### ISSUE THREAD

**Issue by plehegar** [plehegar/transition-notifier#10]
Today, several emails didn't get sent and somehow the tool didn't see the failures. We need to make sure those emails go out....

**Comment by anssiko** [comment]
https://lists.w3.org/Archives/Public/public-review-announce/ has not received any Notifier updates since 14 December 2017.

**Comment by plehegar** [comment]
I don't think we got an issue on this front for a few years. so closing assuming "it just works".
